rust/core-lib/src/editor.rs

```diff
@@ -345,22 +345,31 @@ impl Editor {
                 // backspace deletes max(1, tab_size) contiguous spaces
                 let (_, c) = view.offset_to_line_col(&self.text, region.start);
 
-                let tab_off = c & config.tab_size;
+                // Offset of cursor position from nearest tab
+                let tab_off = c % config.tab_size;
```

Comment:
I think this change alone is the fix?? I have no idea how this was a `&` before, I think it must've been a typo. Changing this to a `%` should enough?